src/vswhere.lib/Utilities.cpp

```diff
@@ -5,9 +5,10 @@
 
 #include "stdafx.h"
 
-std::string to_string(const std::wstring& value)
-{
-    static wstring_converter converter;
```

Comment:
What was wrong with the `wstring_converter` and why didn't you just fix that and use it? If bypassing it, it should be removed so we don't leave cruft in the source.

Reply:
Maybe this was done due to `codecvt_utf8` being deprecated in C++17: https://en.cppreference.com/w/cpp/locale/codecvt_utf8

Reply:
Thanks. In that case, please also remove the typedef above this. Let's clean up fully.

Reply:
The typedef for the converter was already deleted in `Utilities.h`, did I miss anything?

Reply:
Sorry, I missed that. But you have some build breaks to work through. As I suspect, you still need `std::experimental::filesystem` et. al. Just because you changed the compiler version doesn't mean the targeted stdlib changed, nor should it unless required. We want to target min spec. For example, even changing to v143 means that all customers have to have mscvrt143 installed. Can you be sure they do? vswhere has to support all the way back to supported Dev15 versions. That alone may be a breaking change.

Reply:
Sorry, I missed bumping the language standard versions for release build and initially only bumped for debug build, which caused build breaks. I created a test branch to test the build, and I will do a final push once it's verified.

For mscvrt143, we ran `dumpbin /imports` command and verified the list doesn't have importa from mscvrt143. I think we also don't ship vswhere.lib, so I thought it would work out.

Reply:
vswhere.lib is statically linked to vswhere.exe (it's a separate lib so we can test it) so that wasn't a concern, but if you're not seeing MSVC dynamically linked, that's good. I did set it initially to statically link but wasn't sure how many of the MSVC functions that cover.